src/fundus/publishers/au/nine_news.py

```diff
@@ -0,0 +1,35 @@
+import datetime
+from typing import List, Optional
+
+from lxml.cssselect import CSSSelector
+
+from fundus.parser import ArticleBody, BaseParser, ParserProxy, attribute
+from fundus.parser.utility import (
+    extract_article_body_with_selector,
+    generic_author_parsing,
+    generic_date_parsing,
+)
+
+
+class NineNewsParser(ParserProxy):
+    class V1(BaseParser):
+        _paragraph_selector = CSSSelector("div.article__body div.block-content")
```

Comment:
Judging from [this](https://www.9news.com.au/world/usa-jeffrey-epstein-documents-released-but-some-questions-remain-unanswered/141e61e9-8a07-42e4-a527-2f03f45e7ef0) article, the newspaper supports `subheadlines` like "JETSETTER TO CONVICT". Could you add a selector for those? And update the test case using `python -m scripts.generate_parser_test_files -oj` afterward.